lib/govuk_ab_testing/ab_test.rb

```diff
@@ -0,0 +1,23 @@
+module GovukAbTesting
+  class AbTest
+    attr_reader :experiment_name
+
+    def initialize(experiment_name)
+      @experiment_name = experiment_name
+    end
+
+    # Internal name of the header
+    def request_header
+      "HTTP_GOVUK_ABTEST_#{experiment_name.upcase}"
+    end
+
+    def response_header
+      "GOVUK-ABTest-#{cookie_name}"
+    end
+
+    # `example` -> `Example`
+    def cookie_name
+      experiment_name.capitalize
```

Comment:
Feels like we're missing something with the naming - is https://github.com/alphagov/govuk-cdn-config/pull/17 correct?

https://github.com/alphagov/govuk-cdn-config/blob/f8f3cae5cb06570499e7f58e95b4f72df7e7d0cb/vcl_templates/www.vcl.erb#L212